Resources/Prototypes/_Moffstation/Roles/Jobs/CentComm/official.yml

```
@@ -0,0 +1,97 @@
+# Executive - high ranking
+- type: job
+  id: CentralCommandExecutive
+  name: job-name-centcom-executive
+  description: job-description-centcomoff
+  playTimeTracker: JobCentralCommandExecutive
+  setPreference: false
+  startingGear: CentcomGear
+  icon: "JobIconNanotrasen"
+  supervisors: job-supervisors-hos
+  canBeAntag: false
+  accessGroups:
+    - AllAccess
+  access:
+    - CentralCommand
+  special:
+    - !type:AddImplantSpecial
+      implants: [ MindShieldImplant, DeathRattleImplantCentcomm ]
```

similar formatting issues in the rest of the file

```suggestion
  accessGroups:
  - AllAccess
  access:
  - CentralCommand
  special:
  - !type:AddImplantSpecial
    implants: [ MindShieldImplant, DeathRattleImplantCentcomm ]
```